lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -287,9 +287,9 @@ class CategoryRenderer {
       clumpElement.setAttribute('open', '');
     }
 
-    const summaryInnerEl = this.dom.find('.lh-audit-group__summary', clumpElement);
-    const chevronEl = summaryInnerEl.appendChild(this._createChevron());
-    chevronEl.title = Util.i18n.strings.auditGroupExpandTooltip;
+    const summaryInnerEl = this.dom.find('div.lh-audit-group__summary', clumpElement);
+    summaryInnerEl.appendChild(this._createChevron());
+    summaryInnerEl.title = Util.i18n.strings.auditGroupExpandTooltip;
```

Comment:
I looked at https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure and a few other things and i'm convinced we're better completely without this title.

1. none of the examples here or elsewhere have an extra tooltip written as a command, instead they all rely on descriptive text to describe whats behind the disclosure
2. in our impl, it still says `show audits` even after you expand. we probably meant to switch to `hide` but never cared enough.    

let's delete!

-----------

also @connorjclark  you can delete the `title` in the HTML since we now know it's invalid https://github.com/GoogleChrome/lighthouse/blob/dom-find-types/lighthouse-core/report/html/templates.html#L37